frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -611,22 +612,27 @@ const crdCardRowStateToProps = ({ k8s }, { crdDescs }) => {
   };
 };
 
-export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProps) => (
-  <div className="co-crd-card-row">
-    {_.isEmpty(props.crdDescs) ? (
-      <span className="text-muted">No Kubernetes APIs are being provided by this Operator.</span>
-    ) : (
-      props.crdDescs.map((desc) => (
-        <CRDCard
-          key={referenceForProvidedAPI(desc)}
-          crd={desc}
-          csv={props.csv}
-          canCreate={props.createable.includes(referenceForProvidedAPI(desc))}
-        />
-      ))
-    )}
-  </div>
-));
+export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProps) => {
+  const internals = getInternalObjects(props.csv.metadata.annotations);
+  return (
+    <div className="co-crd-card-row">
+      {_.isEmpty(props.crdDescs) ? (
+        <span className="text-muted">No Kubernetes APIs are being provided by this Operator.</span>
+      ) : (
+        props.crdDescs
+          .filter((crd) => !isInternalObject(internals, crd.name))
+          .map((desc) => (
```

Comment:
Suggest using reduce instead of filter -> map.

Reply:
ack